backend/app/common/netmiko/netmiko_client.py

```diff
@@ -84,14 +89,16 @@ def _build_connection_dict(self, ip: str, port: int, uname: str, pwd: str) -> di
     def _exec_download_commands(self, connect_handler: BaseConnection):
         config_result: str = connect_handler.send_command(self.RUNNING_CONFIG_CMD)
         neighbors_result: str = connect_handler.send_command(self.CDP_NEIGHBORS_CMD)
-        return config_result, neighbors_result
+        hostname: str = connect_handler.find_prompt()[:-1]
```

Comment:
A co jak prompt to `R54(config-if)#` ?

Reply:
fakty

Reply:
chociaż wait, czy taka sytuacja w ogóle może mieć miejsce? zwłaszcza jeśli przeniosę to jako pierwsze co ma się wykonać po stworzeniu połączenia i wykonaniu enable

Reply:
twoja decyzja

Reply:
dobra, jak i tak teraz poprawiam resztę to też to ogarnę